services/src/datasets/external/netcdfcf/mod.rs

```diff
@@ -81,7 +79,11 @@ pub struct NetCdfCfDataProviderDefinition {
     pub name: String,
     pub description: String,
     pub priority: Option<i16>,
-    pub path: PathBuf,
+    /// Path were the NetCDF data can be found
+    pub data: PathBuf,
+    /// Database configuration for storing metadata of overviews
+    pub metadata_db_config: DatabaseConnectionConfig,
+    /// Path were overview files are stored
```

Comment:
would it make sense to store the overviews inside the database? This would guarantee consistency with the metadata. Gdal can read from Postgres right?

Reply:
I would not start with this task to store rasters in Postgres.
1. We haven't done it before (for good reasons?).
2. The driver was called WKT Raster and was created in Google summer of code which sounds dubious.